### PR TITLE
fix: dimmer color for Angular and fix .on-scroll on unmount

### PR DESCRIPTION
**Pull request by dozortsev**
## ✅ Checklist

- [ ] Make PR title meaningful and follow [the commit lint format](https://github.com/transferwise/neptune-web/blob/master/CONTRIBUTING.md#versioning-and-commit-lint) (it will appear in the changelog)
- [ ] Changes are tested and all tests pass
- [ ] Changes meet [accessibility standards](https://github.com/transferwise/neptune-web/blob/main/ACCESSIBILITY.md) and there are no violations in the console
- [ ] Changes work in all supported browsers (don't forget IE11)
- [ ] You've updated the documentation if necessary

### CODE REVIEW

File: packages/css/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "@transferwise/neptune-tokens": "^1.0.0",
-    "bootstrap": "github:transferwise/bootstrap#semver:^5.20.0",
+    "bootstrap": "github:transferwise/bootstrap#7.0.0",
```
Comment (dozortsev) [inline packages/css/package.json:45]: just noticed that our BT dependency is slightly off for a few versions

File: packages/css/src/less/modals.less [hunk 2]
```diff
@@ -3,4 +3,184 @@
 @import (reference) '../variables/legacy-variables.less';
 @import 'bootstrap/less/mixins/vendor-prefixes.less';
 @import 'bootstrap/less/mixins/opacity.less';
-@import 'bootstrap/less/modals.less';
+
```
Comment (dozortsev) [inline packages/css/src/less/modals.less:6]: bringing content of [`modals.less`](https://github.com/transferwise/bootstrap/blob/master/less/modals.less) into neptune-css with update color for dimmer

File: packages/docs/package.json [hunk 3]
```diff
@@ -43,7 +43,6 @@
     "@transferwise/icons": "^2.14.0",
     "@transferwise/neptune-css": "^10.0.4",
     "@transferwise/neptune-tokens": "^4.1.0",
-    "bootstrap": "github:transferwise/bootstrap#semver:^5.20.0",
```
Comment (dozortsev) [inline packages/docs/package.json:46]: seems redundant in this package

File: packages/css/src/less/modals.less (hunk 2 above)
Comment (fergusjordantw) [reply packages/css/src/less/modals.less:6]: Are we sure this is the best idea? Not sure we should be bringing styles into a package that isn't actually connected to the component the styles are referencing

Comment (dozortsev) [reply packages/css/src/less/modals.less:6]: > ..Are we sure this is the best idea?.

yes, this is according our Neptune CSS plan

> ..Not sure we should be bringing styles into a package that isn't actually connected to the component the styles are referencing

Neptune CSS is a css for styleguide-components as well

eventually all necessary BT styles will migrate into Neptune CSS, this is better than keep making changes and keep supporting Bootstrap

Comment (fergusjordantw) [reply packages/css/src/less/modals.less:6]: Huh, okay, didn't know that

**Review by fergusjordantw — APPROVED** [review]

**Pull request merged**